test/e2e/discovery/headless_services.go

```diff
@@ -39,41 +39,25 @@ var _ = Describe("[discovery] Test Headless Service Discovery Across Clusters",
 
 	When("a pod tries to resolve a headless service in a remote cluster", func() {
 		It("should resolve the backing pod IPs from the remote cluster", func() {
-			if !framework.TestContext.GlobalnetEnabled {
-				RunHeadlessDiscoveryTest(f)
-			} else {
-				framework.Skipf("Globalnet is enabled, skipping the test...")
-			}
+			RunHeadlessDiscoveryTest(f)
```

Comment:
This PR depends on the Submariner datapath PR which is currently under review - https://github.com/submariner-io/submariner/pull/1456

So we should not merge this until the Submariner PR is merged and references to Submariner are updated in go.mod
@vthapar can you please include "Depends on: https://github.com/submariner-io/submariner/pull/1456" in the commit message.

Reply:
@sridhargaddam This doesn't depend on datapath coz in LH we don't use curl for testing. We use `dig` which only checks if service is resolved, not connectivity.